api/query.py

```diff
@@ -253,3 +267,31 @@ def get_invalid_taint_models(
                 )
             )
     return errors
+
+
+def types(pyre_connection: PyreConnection, modules: Iterable[str]) -> List[Types]:
```

Comment:
This is really a collection of paths, not modules, right?

```suggestion
def types(pyre_connection: PyreConnection, paths: Iterable[str]) -> List[Types]:
```